Lumino/js/GatewayView.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { Alert, Button, FlatList, Text, View } from 'react-native';
+
+const GatewayView = () => {
+  return (
+    <View>
+      <View>
+        <Text>Controllers' list</Text>
```

Comment:
I think it's not even correct gramatically, the list doesn't belong to the controllers, its a list of controllers.